index.html

```diff
@@ -248,9 +248,6 @@ <h3>
           attributes of {{PerformanceEntry}} interface:
         </p>
         <ul>
-          <li>The <code>name</code>
-          getter steps are to return the [=current document=]'s {{Document/URL}}.
-          </li>
```

Comment:
Maybe add a note indicating that this has a getter in the RT spec? your call

Reply:
I don't think it's necessary. We use so many other getters from the RT spec